patches/minecraft/net/minecraft/client/renderer/entity/ItemRenderer.java.patch

```diff
@@ -57,6 +57,14 @@
              crashreportcategory.m_128165_("Item Damage", () -> {
                 return String.valueOf(p_174237_.m_41773_());
              });
+@@ -323,6 +_,7 @@
+ 
+    public void m_115174_(Font p_115175_, ItemStack p_115176_, int p_115177_, int p_115178_, @Nullable String p_115179_) {
+       if (!p_115176_.m_41619_()) {
++         if(net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.client.event.RenderGuiItemDecorationsEvent.Pre(p_115175_, p_115176_, p_115177_, p_115178_, p_115179_))) return;
```

Comment:
Create a method in ForgeHooksClient, instead of constructing the event and firing it yourself.